data-prepper-plugins/s3-source/README.md

```diff
@@ -70,9 +70,9 @@ All Duration values are a string that represents a duration. They support ISO_86
 
 * `s3_select` : S3 Select Configuration.
 
-* `query_statement` : Provide s3 select query to process the data using S3 select for the particular bucket.
+* `expression` : Provide s3 select query to process the data using S3 select for the particular bucket.
 
-* `data_serialization_format` : Provide the s3 select file format (csv/json/Apache Parquet) Amazon S3 uses this format to parse object data into records and returns only records that match the specified SQL expression. You must also specify the data serialization format for the response.
+* `input_serialization` : Provide the s3 select file format (csv/json/Apache Parquet) Amazon S3 uses this format to parse object data into records and returns only records that match the specified SQL expression. You must also specify the data serialization format for the response.
```

Comment:
Please specify required or optional for these parameters similar to what is done for the other S3 source parameters